multimedia/welle.io/Portfile

```diff
@@ -29,30 +30,28 @@ qt5.depends_component   qtcharts \
                         qtquickcontrols2
 qt5.min_version         5.10.0
 
-depends_lib-append      port:faad2 \
+depends_lib-append      port:airspy \
+                        port:faad2 \
                         port:fftw-3-single \
                         port:lame \
                         port:libusb \
-                        port:mpg123
+                        port:mpg123 \
+                        port:rtl-sdr \
+                        port:SoapySDR
 
-default_variants        +rtlsdr +soapysdr +airspy
-
-variant rtlsdr description {Add RTL-SDR support} {
-    configure.pre_args-append \
-        -DRTLSDR=ON
-    depends_lib-append port:rtl-sdr
+variant no_rtlsdr description {Disable RTL-SDR support} {
+    configure.pre_args-replace -DRTLSDR=ON -DRTLSDR=OFF
+    depends_lib-delete port:rtl-sdr
 }
 
-variant soapysdr description {Add SoapySDR support} {
-    configure.pre_args-append \
-        -DSOAPYSDR=ON
-    depends_lib-append port:SoapySDR
+variant no_soapysdr description {Disable SoapySDR support} {
+    configure.pre_args-replace -DSOAPYSDR=ON -DSOAPYSDR=OFF
+    depends_lib-delete port:SoapySDR
```

Comment:
You are going the wrong way here. You need to restore the variant. We are not using "negative" variant names for consistency.